cat/CaT-service.yaml

```diff
@@ -554,27 +554,25 @@ paths:
             - buyer
       summary: Create Project from an Agreement (SCC-440)
       
-  /tenders/projects/{proc-id}/confirm:
+  /tenders/projects/{proc-id}/name:
     parameters:
       - $ref: '#/components/parameters/ProcurementParam'
-    post:
-      deprecated: true
-      description: Confirm the details transfered or created for the agreement are what the user wants.
+    put:
+      deprecated: false
+      description: Replace the name transfered or created for the procurement automatically with what the user wants.
```

Comment:
Very minor observation - `transfered` should be `transferred`

Reply:
Corrected and release to the branch